### ISSUE TITLE
S3 Interoperability

### ISSUE THREAD

**Issue by arik-mlvx** [fsouza/fake-gcs-server#1330]
My company uses s5cmds to speed up uploads to GCS using the S3 interoperability feature. Is S3 interoperability supported or planned to be supported?

**Comment by fsouza** [comment]
Yes, there's a PR open (Google doesn't call it S3 compatibility, but rather XML API), but tests are failing. I haven't had a chance to look into it just yet
